src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -36,6 +50,115 @@
+      // Running on a local server requires storing API key in environment variable.
+      // In this case, uncomment the following line, and comment out the call to
+      // accessSecretVersion. Do the reverse when deploying.
+      // String electionApiKey = System.getenv("GOOGLE_API_KEY");
+
+      String electionApiKey = getApiKey("112408856470", "election-api-key", "1");
+      URL url = new URL(baseURL + electionApiKey);
+      conn = (HttpURLConnection) url.openConnection();
+
+      conn.setRequestMethod("GET");
+      conn.setRequestProperty("Accept", "application/json");
+
+      if (conn.getResponseCode() != HttpURLConnection.HTTP_OK) {
+        throw new RuntimeException(
+            "HTTP GET Request Failed with Error code : " + conn.getResponseCode());
+      }
+
+      // Using IO Stream with Buffer for increased efficiency
+      reader = new BufferedReader(new InputStreamReader(conn.getInputStream(), "utf-8"));
+      String output = null;
+
+      while ((output = reader.readLine()) != null) {
+        strBuf.append(output);
+      }
+    } catch (MalformedURLException e) {
+      response.setContentType("text/html");
+      response.getWriter().println("URL is incorrectly formatted");
+      return;
+    } catch (IOException e) {
+      response.setContentType("text/html");
+      response.getWriter().println("Cannot retrieve information from provided URL");
+      return;
+    } finally {
+      if (reader != null) {
+        try {
+          reader.close();
+        } catch (IOException e) {
+          logger.log(Level.WARNING, e.getMessage());
+        }
+      }
+      if (conn != null) {
+        conn.disconnect();
+      }
+    }
+
+    String elections = strBuf.toString();
+
+    JSONObject obj = new JSONObject(elections);
+    JSONArray electionData = obj.getJSONArray("elections");
+    int numElections = electionData.length();
+
+    for (int i = 0; i < numElections; ++i) {
+      JSONObject currentElection = electionData.getJSONObject(i);
+      Entity electionEntity = new Entity("Election");
+
+      /* The "id" of an Election Entity is stored as a property instead of
+       * replacing the Datastore-generated ID because  Datastore may
+       * accidentally reassign IDs to other entities. To avoid this problem, I would have
+       * to obtain a block of IDs with allocateIds(), but this is also difficult because
+       * election IDs are not always consecutive numbers and other entities we plan to store
+       * in Datastore will not have IDs from the Civic Information API (ex. policies) */
+      electionEntity.setProperty("id", currentElection.getLong("id"));
+      electionEntity.setProperty("name", currentElection.getString("name"));
+      electionEntity.setProperty("scope", currentElection.getString("ocdDivisionId"));
+      electionEntity.setProperty("date", currentElection.getString("electionDay"));
+
+      datastore.put(electionEntity);
+    }
+
+    Gson gson = new Gson();
```

Comment:
Do you set the response just for the sake of debugging? Also comment that you're filling out the response just for debugging purposes if so. 

Could consider just logging instead.